docker/utils/utils.py

```diff
@@ -44,6 +44,25 @@
 }
 
 
+def create_ipam_pool(subnet=None, iprange=None, gateway=None,
+                     aux_addresses=None):
+    return {
+        'subnet': subnet,
+        'iprange': iprange,
+        'gateway': gateway,
+        'auxaddresses': aux_addresses
+    }
+
+
+def create_ipam_config(driver='default', pool_configs=None):
+    if not pool_configs:
+        pool_configs = []
+    return {
+        'driver': driver,
+        'config': pool_configs
```

Comment:
`pool_configs or []` would allow us to get rid of the first two lines here. Small readability improvement.